src/util/messages.py

```diff
@@ -47,3 +47,7 @@ def added_favorites_msg(favorites):
 
 def menu_has_favorite_msg(favorite):
     return f'Hey! This meal contains {favorite}'
+
+
+def notification_view_msg():
```

Comment:
this should take in the meal?

Reply:
nope. This is the header text for the notification menu which displays both meals